docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -155,17 +155,32 @@ can be used only through tsh in [a local terminal tab](#opening-a-local-terminal
 Alternatively, you can look for the application in the search bar and press `Enter` to launch it in
 the browser.
 
+### Connecting to TCP apps with VNet on macOS
+
+VNet is a virtual network emulation tool available in Teleport Connect. It enables any program on
+your computer to connect to TCP applications protected by Teleport, with no extra changes required
+to the program itself. It removes the need to manually create tunnels for TCP applications.
+
+1. Log in to the cluster.
+1. To start VNet, open the connection list in the top left and click the connect icon to the right of
+   VNet.
+1. Now access any of your web and TCP apps over its regular public address.
+
+VNet automatically authenticates the connection with your user certificate. See [Using
+VNet](vnet.mdx) for more details.
```

Comment:
What kinds of details would the user expect to find here?

Reply:
I guess just more information about VNet than those couple of sentences that we have here perhaps? Do you think we should remove the link or just reword this sentence?

Reply:
Looking again, I think it's fine as it is since this is a brief conceptual guide with how-to instructions mixed in.